app/routes/popup/Transaction/components/Transaction/FooterTransactions/index.js

```diff
@@ -49,20 +54,33 @@ class FooterTransactions extends Component {
     }
 
     return (
-      <span className={styles.buttonGroup}>
-        <button
-          onClick={this.handleRejectTransaction}
-          className={cx(styles.button, styles.reject)}
-        >
-          REJECT
-        </button>
-        <button
-          onClick={this.handleConfirmTransaction}
-          className={cx(styles.button, styles.confirm)}
-        >
-          CONFIRM
-        </button>
-      </span>
+      <Network
+        render={({ online }) =>
+          online
+            ? (loadedData && !reviewedTx &&
+              <span className={styles.buttonGroup}>
```

Comment:
This is a four-level depth component render. I suggest extracting into local components the logic. 

Something like: 
```
<Network
     render={({ online }) => online
        ? <ReviewTx loadedData={loadedData} reviewedTx={reviewedTx} />
        : <NetworkError />
/>
```
Also, what happens if ```reviewedTx``` is true? How is the component's behaviour?